Expo1900/Expo1900/View/ExpoMenuTableViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ExpoMenuTableViewCell.swift
+//  Expo1900
+//
+//  Created by 재재, 보리사랑 on 2022/06/17.
+//
+
+import UIKit
+
+class ExpoMenuTableViewCell: UITableViewCell {
+    @IBOutlet weak var expoItemImageView: UIImageView!
+    @IBOutlet weak var expoItemNameLabel: UILabel!
+    @IBOutlet weak var expoItemDescriptionLabel: UILabel!
+    
+    func setupImageView(name: String) {
+        self.expoItemImageView.image = UIImage(named: name)
+    }
+    
+    func setupLableView(name: String, shortDescription: String) {
+        self.expoItemNameLabel.text = name
+        self.expoItemDescriptionLabel.text = shortDescription
+    }
```

Comment:
```suggestion
    func setupUI(with expoItemElement: ExpoItemElement) {
         setupLableView(name: expoItemElement.name, shortDescription: expoItemElement.shortDesc)
         setupImageView(name: expoItemElement.imageName)
    } 

    private func setupImageView(name: String) {
        self.expoItemImageView.image = UIImage(named: name)
    }
       
    private func setupLableView(name: String, shortDescription: String) {
        self.expoItemNameLabel.text = name
        self.expoItemDescriptionLabel.text = shortDescription
    }
```
=> 이렇게 모델 값을 전달 인수로 넘겨 외부에서 한번만(setupUI만) 호출하는 방법도 있습니다. 이러면 외부에서는 setupUI(with:)만 호출하고 cell 코드 안쪽에서만 코드가 변경하면 되는 장점이 있습니다.